examples/notebooks/get-started/learn_beam_basics_by_doing.ipynb

```diff
@@ -0,0 +1,1068 @@
+        "  | 'Second step' >> step2\n",
+        "  | 'Third step' >> step3\n",
+        ")\n",
+        "```"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "GXQ__Kwxvr3J"
+      },
+      "source": [
+        "## 1.2 Loading Our Data\n",
+        "\n",
+        "Now, you can try to write your own pipeline!\n",
+        "\n",
+        "First, let's load the example data we will be using throughout this tutorial into our file directory. This [dataset](https://archive.ics.uci.edu/ml/datasets/SMS+Spam+Collection) consists of a **collection of SMS messages in English tagged as either \"spam\" or \"ham\" (a legitimate SMS\n",
+        ")**.\n",
+        "\n",
+        "For this tutorial, we will create a pipeline to **explore the dataset using Beam to count words in SMS messages that contain spam or ham**."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "BkZ0wxsBPuvO"
+      },
+      "outputs": [],
+      "source": [
+        "# Creates a data directory with our dataset SMSSpamCollection\n",
+        "!mkdir -p data\n",
+        "!gsutil cp gs://apachebeamdt/SMSSpamCollection data/"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TiHFDIMnLTbm"
+      },
+      "source": [
+        "**What does the data look like?**\n",
+        "\n",
+        "This dataset is a `txt` file with 5,574 rows and 4 columns recording the following attributes:\n",
+        "1. `Column 1`: The label (either `ham` or `spam`)\n",
+        "2. `Column 2`: The SMS as raw text (type `string`)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "! head data/SMSSpamCollection"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "r-0WM38KNaTI"
+      },
+      "source": [
+        "## 1.3 Writing Our Own Pipeline\n",
+        "\n",
+        "Now that we understand our dataset, let's go into creating our pipeline.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1Ajz7-WQCVfj"
+      },
+      "source": [
+        "To initialize a `Pipeline`, you first assign your pipeline `beam.Pipeline()` to a name. Assign your pipeline to the name, `pipeline`, in the code cell below."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "cXm0n80E9ZwT"
+      },
+      "outputs": [],
+      "source": [
+        "#@title Edit This Code Cell\n",
+        "..."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "cellView": "form",
+        "id": "q0gzEcos9hjp"
+      },
+      "outputs": [],
+      "source": [
+        "#@title Answer\n",
+        "pipeline = beam.Pipeline()"
```

Comment:
I noticed this as well and tried to hide this from the users. Since you noticed this as well, I added `pipeline = beam.Pipeline()`  explicitly to each cell.